### ISSUE TITLE
Ruby support broken in 8.1.873 (154) on Mojave

### ISSUE THREAD

**Issue by RubenVerborgh** [macvim-dev/macvim#847]
Executing `ruby 1` gives

```
E448: Could not load library function rb_ary_detransient
E266: Sorry, this command is disabled, the Ruby library could not be loaded. 
```

**Comment by ychin** [comment]
Yeah this MacVim binary release upgraded the ruby version. Glad you got it working.

**Comment by vnazarenko** [comment]
By the way, I'm getting this error for all other ruby versions, to fix that i switched to 2.6.1
